### PR TITLE
Just adding total downloads badge :v

### PR DESCRIPTION
**Pull request by LyQuid12**
I know Geno, I'm still learning the Twitter API. Just wait, I'll definitely contribute ;)

~ LyQuid <3

### CODE REVIEW

**Comment by LyQuid12** [comment]
Hmmmm, 69

**Comment by LyQuid12** [comment]
Sussy number

**Comment by LyQuid12** [comment]
I'm only added a badge to readme bruh

**Comment by TheGenocides** [comment]
> I'm only added a badge to readme bruh

Well yeah, delete the old downloads badge.

**Comment by LyQuid12** [comment]
> > I'm only added a badge to readme bruh
> 
> Well yeah, delete the old downloads badge.

Hmmm, kk

**Comment by LyQuid12** [comment]
Done

**Review by TheGenocides — APPROVED** [review]
Merging...

**Comment by LyQuid12** [comment]
Yessss sirr

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 <img alt="PyPI - License" src="https://img.shields.io/pypi/l/PyTweet">
 
+<img alt="Total Downloads" src="https://pepy.tech/badge/pytweet">
```
[the remark this answers is not in the document]
Comment (LyQuid12) [reply README.md:9]: Not really

**Comment by kaylebetter** [comment]
sussy

**Comment by LyQuid12** [comment]
> sussy

Baka

**Pull request merged**